tests/src/drivers/key_management.c

```diff
@@ -59,7 +63,8 @@ psa_status_t test_transparent_generate_key(
     }
 
     /* Copied from psa_crypto.c */
-#if defined(MBEDTLS_ECP_C)
+#if defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_KEY_PAIR) || \
+    defined(MBEDTLS_PSA_BUILTIN_KEY_TYPE_ECC_PUBLIC_KEY)
```

Comment:
This code is to simulate an accelerator driver thus the guards should rather be the ACCEL ones.